src/components/Graph.js

```diff
@@ -1,33 +1,39 @@
+/*
```

Comment:
I added Radium to this component as well.  Small amount of styling is used as the actual graph generated is a canvas object.